cfg_files/stanford/fp30_smurf_startup.cfg

```diff
@@ -3,7 +3,7 @@ shelfmanager=shm-smrf-sp01
 set_crate_fans_to_full=true
 # COMTEL max fan level is 100, ASIS is 15, ELMA is 15
 ## COMTEL in RF lab
-max_fan_level=15
+max_fan_level=10
```

Comment:
@swh76 The comment says the max fan leve is 100, but the var is 10

Reply:
Max for ELMA is 15 which is what we have on campus, this is correct albeit not at all obvious.  JIRA exists for auto detecting crate manufacturer to make this less arbitrary.

Reply:
Do we want to change the comment? I'll approve the PR either way.